### ISSUE TITLE
Toggles, radios and checkboxes in Element settings don't render properly when turned on

### ISSUE THREAD

**Issue by bblacher** [element-hq/element-web#27166]
### Steps to reproduce

Open settings while using dark theme

### Outcome

![image](https://github.com/element-hq/element-web/assets/42916911/10073204-a371-4f3b-a960-7b8a14575b6a)

### Operating system

Arch Linux rolling

### Application version

Element version: 1.11.60 Crypto version: Rust SDK 0.7.0 (b1918e9), Vodozemac 0.5.1

### How did you install the app?

https://archlinux.org/packages/extra/x86_64/element-desktop/ (modified PKGBUILD to use electron 29 and newest Element version)

### Homeserver

Synapse 1.102.0

### Will you send logs?

No

**Comment by wrenix** [comment]
Also on
1. light mode
2. checkbox (like on theming for light and dark mode)

**Comment by bblacher** [comment]
Screenshots showing the problem on light theme:
![image](https://github.com/element-hq/element-web/assets/42916911/a16ed096-86a8-44a9-925a-6258bcd485e0)

Missing checkbox:
![image](https://github.com/element-hq/element-web/assets/42916911/0ee67bdf-dc54-41bd-bb0d-97c784831365)

**Comment by wrenix** [comment]
Looks like the Playwright test does not work correct:
https://github.com/matrix-org/matrix-react-sdk/blob/v3.94.0/playwright/snapshots/settings/preferences-user-settings-tab.spec.ts/Preferences-user-settings-tab-should-be-rendered-properly-1-linux.png

in https://github.com/matrix-org/matrix-react-sdk/pull/12299

---

PS: I use the docker-image

**Comment by florianduros** [comment]
@wrenix It's working correctly.

Following my analysis, the bug is only on the built version of the desktop app.

**Comment by florianduros** [comment]
We are missing in the built some compound variables like: `--cpd-color-bg-accent-rest`

**Comment by mokazemi** [comment]
> @wrenix It's working correctly.
> 
> Following my analysis, the bug is only on the built version of the desktop app.

No. I'm so experiencing it on the web version.

**Comment by t3chguy** [comment]
@florianduros I can repro on app & staging

![image](https://github.com/element-hq/element-web/assets/2403652/c48f3279-5e86-4a26-989a-cd102fa60328)
![image](https://github.com/element-hq/element-web/assets/2403652/345ae80f-4fd3-4526-984e-35e054ca1bbc)

**Comment by florianduros** [comment]
@t3chguy Yep I saw it in the evening. I'm trying to update all the old design-tokens reference to fix it.

**Comment by florianduros** [comment]
Should be fixed in the next release.

**Comment by wrenix** [comment]
works, thanks
